src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -202,7 +208,7 @@ const styles = StyleSheet.create({
   },
   container: {
     backgroundColor: COLORS.default_signed_secondary,
-    borderRadius: 15,
+    borderRadius: 16,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The slight adjustment to the `borderRadius` property is a minor visual change that doesn't affect the functionality of the component. However, it's important to ensure consistency in design across the application.
```diff
- borderRadius: 15,
+ borderRadius: 16,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->